clients/gtest/trtri_gtest.cpp

```diff
@@ -47,7 +50,8 @@ namespace
     enum trtri_kind
     {
         trtri_k,
-        trtri_batched_k
+        trtri_batched_k,
+        trtri_strided_batched_k
```

Comment:
Use trailing comma on the last entry so that all entries are uniform and can be expanded later without changing the last line.